src/devdox_ai_git/git_managers.py

```
@@ -0,0 +1,388 @@
+            repos_page = repos_paginated.get_page(page - 1)
+
+            pagination_info = GitHubManager.get_pagination_info(
+                total_count=repos_paginated.totalCount,
+                page=page,
+                per_page=per_page,
+            )
+
+            return {
+                "repositories": repos_page,
+                "pagination_info": pagination_info,
+            }
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)